frodo.js

```diff
@@ -5,78 +5,78 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.